boards/sltb001a/Makefile.dep

```diff
@@ -0,0 +1,5 @@
+ifneq (,$(filter saul_default,$(USEMODULE)))
+    USEMODULE += saul_gpio
+    USEMODULE += bmp280
+    USEMODULE += si70xx
```

Comment:
once #7296 is merged, here you can use the exact reference of the sensor (si7021 I think)

Reply:
And included :-)